Makefile

```diff
@@ -1,15 +1,15 @@
 # ====================================================================================
 # Setup Project
 
-PROJECT_NAME := provider-jet-template
+PROJECT_NAME := provider-jet-alicloud
```

Comment:
`provider-jet-alicloud` or `provider-jet-alibaba` ?
Should we change repo url if we want to go `provider-jet-alicloud` ?